packages/flutter/lib/src/material/list_tile.dart

```diff
@@ -232,6 +232,25 @@ enum ListTileControlAffinity {
 ///
 /// {@tool snippet}
 ///
+/// This example uses an [Expanded] within a [Row] to demonstrate how to
+/// overcome the issue with a [ListTile] attempting to expand horizontally
+/// without proper constraints.
+///
+/// ```dart
+/// Row(
+///   children: const <Widget>[
+///     Expanded(
+///       child: ListTile(
+///         leading: FlutterLogo(),
+///         title: Text('The ListTile is expanded to fill the available space'),
+///       ),
+///     ),
+///   ],
+/// )
+/// ```
+/// {@end-tool}
+/// {@tool snippet}
+///
```

Comment:
Could we move this to under the first example? The first example is a generic set of use-cases for `ListTile`, and this particular API doc addresses a unique use-case.